app/models/method_category.rb

```diff
@@ -11,7 +11,7 @@
 class MethodCategory < ActiveRecord::Base
   validates :name, presence: true, length: { maximum: 255,
             too_long: "%{count} is the maximum character length."}
-
+  attr_accessible :name, :characteristic_groups
```

Comment:
characteristic_groups isn't a database field on MethodCategory, you can remove that